easybuild/easyblocks/o/openfoam.py

```diff
@@ -192,6 +197,11 @@ def configure_step(self):
         # parallel build spec
         env.setvar("WM_NCOMPPROCS", str(self.cfg['parallel']))
 
+        # OpenFOAM >= 3.0.0 can use 64 bit integers
+        if 'extend' not in self.name.lower() and LooseVersion(self.version) >= LooseVersion('3.0'):
+            if self.toolchain.options['i8']:
+                env.setvar("WM_LABEL_SIZE", '64')
```

Comment:
@boegel If we do not set, it defaults to 32 bit. But you are right, I should be consistent.